packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -186,7 +184,7 @@ export function PrivateBlockToolbar( {
 	// Shifts the toolbar to make room for the parent block selector.
 	const classes = clsx( 'block-editor-block-contextual-toolbar', {
 		'has-parent': showParentSelector,
-		'is-inverted-toolbar': isNavigationMode && ! hasFixedToolbar,
+		'is-inverted-toolbar': isContentOnlyMode && ! hasFixedToolbar,
```

Comment:
`contentOnly` is also used outside of navigation (write) mode:
- block overrides in synced patterns
- templateLock `contentOnly`
- Post content blocks when editing a post with the 'Show Template'

Those all now have the inverted toolbar after this change, whereas my understanding is that it's only for navigation mode to indicate the different mode.

Before:
<img width="481" height="282" alt="Screenshot 2025-08-08 at 5 37 13 pm" src="https://github.com/user-attachments/assets/98787d25-f527-484a-92ff-48aa796b399c" />

After:
<img width="483" height="278" alt="Screenshot 2025-08-08 at 5 37 36 pm" src="https://github.com/user-attachments/assets/07cd4770-9057-4439-adfa-6be5b3fe97d3" />

Reply:
Ok, that's fine if so.

There might be some contrast issues to work out with the purple icon color (also not sure about the grey text color).